[안혜수] 마이페이지 구현 완료

hyesoo-ahn opened the pull request:
## 수정 사항 간략한 한줄 요약
- 페이지 레이아웃
- 목데이터로 데이터 받아오기
- 동적라우팅
		 
## 수정 사항들 자세한 내용
- 페이지 레이아웃
- 목데이터로 데이터 받아오기
- 동적라우팅

## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.
- [x] Git rebase와 squash를 했고 커밋 수가 하나 인것을 확인 했습니다.

claire-euni commented on src/Pages/MyPage/CreateClassList.js:
```diff
@@ -0,0 +1,158 @@
+  render() {
+    const { created } = this.props;
+
+    // console.log(created);
```
콘솔 삭제!

claire-euni commented on src/Pages/MyPage/CreateClassList.js:
```diff
@@ -0,0 +1,158 @@
+          {created.map((item, index) => (
+            <VideoCard key={index}>
+              <div className='videoBox'>
+                <img src={item.thumbnail} alt='섬네일' />
```
어떤 정보인지 alt는 상세하게 써주세요

claire-euni commented on src/Pages/MyPage/CreateClassList.js:
```diff
@@ -0,0 +1,158 @@
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+const VideoCard = styled.div`
```
혜수님 스타일 컴포넌트 세분화 해주세요!

claire-euni commented on src/Pages/MyPage/HistoryCardList.js:
```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { withRouter } from 'react-router-dom';
```
import 순서 고려해서 다시 수정!

claire-euni commented on src/Pages/MyPage/HistoryCardList.js:
```diff
@@ -0,0 +1,159 @@
+import { css } from 'styled-components';
+import { withRouter } from 'react-router-dom';
+
+class HistoryCardList extends Component {
```
CreateClassList랑 구조가 동일해보이는데 하나의 컴포넌트에서 관리하면 되지 않을까요?

claire-euni commented on src/Pages/MyPage/MyPage.js:
```diff
@@ -5,79 +5,100 @@ import { RiCoupon2Line } from 'react-icons/ri';
 import { AiOutlineHeart } from 'react-icons/ai';
 import { RiShoppingBasket2Line } from 'react-icons/ri';
 import { RiKakaoTalkFill } from 'react-icons/ri';
+import HistoryCardList from './HistoryCardList';
+import CreateClassList from './CreateClassList';
 import VideoCardList from './VideoCardList';
 import Nav from '../../Components/Nav/Nav';
```
이 부분도 순서 다시 수정해주세요 그리고 import와 component 선언 사이에 엔터간격 한 줄 주세요

claire-euni reviewed (commented):
혜수님 스타일 컴포넌트화하는거 한번 해보시면 좋을것 같습니다. 더 세부적으로 나눠주세요~

hyesoo-ahn committed:
resolve

hyesoo-ahn committed:
# This is a combination of 4 commits.
# This is the 1st commit message:

resolve

# This is the commit message #2:

fix

# This is the commit message #3:

fix

# This is the commit message #4:

# This is a combination of 6 commits.
# This is the 1st commit message:

fix

# This is the commit message #2:

add: classdetail페이지

# This is the commit message #3:

classRightSide

# This is the commit message #4:

add: commentInput

# This is the commit message #5:

wip

# This is the commit message #6:

wip

hyesoo-ahn committed:
resolve
